package.json

```diff
@@ -42,7 +43,9 @@
     "test": "mocha --full-trace --check-leaks",
     "precover": "npm run lint && npm run beautify-lint",
     "cover": "istanbul cover node_modules/mocha/bin/_mocha",
-    "travis": "npm run cover -- --report lcovonly"
+    "travis": "npm run cover -- --report lcovonly",
+    "release:create": "standard-version -i CHANGELOG.md -m \"Creating release %s\"",
+    "release:publish": "git push --follow-tags origin master; npm publish"
```

Comment:
@wardpeet - Per the [published documentation](https://webpack.js.org/development/release-process/) we only need a single task `"release": "standard-version"`

From that we can chain commands on to publish different targets. You also don't need to specify the changelog.md nor is the message really necessary.

The publish shouldn't be done as an npm script, it eats publishing errors. We also don't always publish to `dist-tag`. 

In theory using an npm script could be viable for pushing tags to origin but in reality, I can copy and paste the printed command `git push --follow-tags origin master` the library spits out faster than I can type `npm run release:publish`